pe_file/pe_file.py

```diff
@@ -148,7 +148,7 @@ def get_pe_info(self, lcid):
                 if sname:
                     if isinstance(sname, bytes):
                         method = chardet.detect(sname).get('encoding', 'utf-8')
-                        sname = sname.decode(method)
+                        sname = sname.decode(method) if method else str_safe(sname.decode('utf-8'))
```

Comment:
dont decode before safe_str this can crash. safe_str is built to handle both str and unicodes.